lib/tests/tests/functions/flatten/simple.vrl

```diff
@@ -0,0 +1,4 @@
+# object: { "a": { "b": { "c": { "d": 1, "e": 2 }, "f": 3 }, "g": 4 } }
+# result: { "a.b.c.d": 1, "a.b.c.e": 2, "a.b.f": 3, "a.g": 4 }
+
+flatten(.)
```

Comment:
Maybe this test is redudant with the ones from the examples
https://github.com/vectordotdev/vrl/blob/d3a7dd7fda91f6d200ec3d09b70bc20c5af4867f/src/stdlib/unflatten.rs#L96

What do you think? Should I only leave the `unflatten(flatten(.))` one?

Reply:
I think it is still a useful test to assert the unflattened representation. One could imagine a bug that causes round tripping through `flatten` to pass while the output of the `unflatten` call is incorrect.